automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/LibraryData.scala

```diff
@@ -41,5 +41,7 @@ object LibraryData {
       "library:useLimitationOption"->"orsp",
       "library:orsp"->"some orsp")
 
+
```

Comment:
In general we try to avoid adding whitespace in PR's unless it makes things easier to understand.